src/sidebar/util/fetch-config.js

```diff
@@ -132,9 +132,58 @@ async function fetchConfigRpc(appConfig, parentFrame, origin) {
     [],
     3000
   );
-  return fetchConfigEmbed(appConfig, remoteConfig);
+  const mergedConfig = fetchConfigEmbed(appConfig, remoteConfig);
+  return mutateAsyncGroups(mergedConfig, parentFrame, origin);
```

Comment:
Just a thought - rather than passing down all the params for an RPC call to `mutateAsyncGroup`, it might be useful to package all that up into a closure, eg:

```js
const rpcCall = (method, args = [], timeout = 3000) => postMessageJsonRpc.call(
  parentFrame, origin, method, args, timeout
);

const remoteConfig = await rpcCall('requestConfig');
const mergedConfig = fetchConfigEmbed(appConfig, remoteConfig);
mutateAsyncGroups(mergedConfig, rpcCall);
```

Reply:
sure!